terraform/modules/proxmox/cloudinit/bootloader.sh

```
@@ -17,15 +17,15 @@ function error() {
 
   curl -X POST                                                                                                                      \
        -H "Content-Type: application/json"                                                                                          \
-       -d "{\"chat_id\": \"${telegram_chat_id}\", \"text\": \"[${domain}]: $1 in ${SCRIPT}\", \"disable_notification\": false}"    \
+       -d "{\"chat_id\": \"${telegram_chat_id}\", \"text\": \"[${domain}]: $1 in ${SCRIPT}\", \"disable_notification\": false}"     \
```

Shellcheck (0.11.0)

[warning] domain is referenced but not assigned.

(SC2154)